aioshelly/ble/const.py

```diff
@@ -10,31 +10,63 @@
 VAR_EVENT_TYPE = "%event_type%"
 VAR_ACTIVE = "%active%"
 VAR_VERSION = "%version%"
-VAR_INTERVAL_MS = "%interval_ms%"
-VAR_WINDOW_MS = "%window_ms%"
-VAR_DURATION_MS = "%duration_ms%"
-
-DEFAULT_INTERVAL_MS = 320
-DEFAULT_WINDOW_MS = 30
-DEFAULT_DURATION_MS = -1
```

Comment:
Those are currently used by HA integration: `shelly/bluetooth/__init__.py`

Reply:
Yeah, sorry, I thought the default config could be used, but yeah. I just bring them back.